.eslintrc.js

```diff
@@ -1,16 +1,14 @@
-const prettierOptions = require("./.prettier.js");
-
 module.exports = {
-  extends: ["standard", "prettier", "prettier/standard"],
+  extends: ["standard", "plugin:prettier/recommended"],
```

Comment:
so does this automatically pull in the prettierrc?

Reply:
As I understand it, `eslint-plugin-prettier` does report prettier problems as eslint errors, based on the `.prettierrc` config file.

While `eslint-config-prettier` turn off all other rules that would conflict with prettier. 

Using `recommended` does set `prettier/prettier` rule to "error" (i.e. it can be omitted), and it extends `config` with the actual `.prettierrc` configuration.